extensions-contrib/azure-extensions/src/main/java/io/druid/storage/azure/AzureDataSegmentPuller.java

```diff
@@ -58,15 +66,29 @@ public AzureDataSegmentPuller(
     try {
       prepareOutDir(outDir);
 
-      final ByteSource byteSource = new AzureByteSource(azureStorage, containerName, blobPath);
+      log.info(StringUtils.format(
```

Comment:
Don't need `StringUtils.format(` here, `log.info()` could accept exactly the same args.